src/tainting/Dataflow_tainting.ml

```diff
@@ -35,6 +35,13 @@ module Effect = Shape_and_sig.Effect
 module Effects = Shape_and_sig.Effects
 module Signature = Shape_and_sig.Signature
 
+(* Domain-local storage for constructor instance variable taint *)
+let constructor_instance_vars : (string, Lval_env.t) Hashtbl.t Domain.DLS.key =
+  Domain.DLS.new_key (fun () -> Hashtbl.create 16)
```

Comment:
Probably needs to be cleaned after each target to avoid leftover info spilling into next targets.